cocos/ui/editbox/edit-box-impl.ts

```diff
@@ -229,11 +227,6 @@ export class EditBoxImpl extends EditBoxImplBase {
             return;
         }
 
-        if (this.__fullscreen) {
-            view.enableAutoFullScreen(false);
-            // eslint-disable-next-line @typescript-eslint/no-floating-promises
-            screen.exitFullScreen();
```

Comment:
how do you adapt this behavior in PAL ?

Reply:
it is because we've deprecated `view.enableAutoFullScreen`, and its value is false by default
so as for EditBox， we say that the game is not on fullscreen

### what if the developers call screen.requestFullScreen ?
I did some test on Chrome, there is no need to exit full screen if we call `documnet.documentElement.requestFullscreen()`, while we call `canvas.requestFullscreen()` for now.

I think `view.enableAutoFullScreen` is a web only interface (need to be deprecated), and we should only provide `screen.requestFullScreen(): Promise` interface in the future